tests/data/diagnosis_summary.json

```diff
@@ -1,5 +1,298 @@
 [
```

Comment:
What's the goal to revise it?

Reply:
to pass the test for ‘save raw data of all metrics rather than metrics defined in diagnosis_rules.yaml when output_all is True’